slidez-fe-core/src/pages/update/UpdatePage.tsx

```diff
@@ -41,14 +33,50 @@ export const UpdatePage: FC = () => {
         setIsActive(!isActive)
     }
 
-    const [userData, setUserData] = useState(userValues)
-    const [userPassword, setUserPassword] = useState(pwdValues)
+    const initialValues: User = {
+        id: '',
+        firstName: '',
+        lastName: '',
+        email: '',
+        password: '',
+    }
 
-    const handleInfoSubmit = (values: typeof userValues) => {
-        console.log(values)
+    const [userData, setUserData] = useState<User>(initialValues)
+
+    const schema = yup.object({
+        email: yup.string().email().max(64),
+        firstName: yup.string().required().min(3).max(30),
+        lastName: yup.string().required().min(3).max(30),
+    })
+    const currentToken =
```

Comment:
Tokens must **NEVER** leak into repository

Reply:
Agree, but they are temporary + only for local server, so it's not a big deal. But get it from localstorage